src/sentry/runner/decorators.py

```diff
@@ -36,20 +36,31 @@ def inner(ctx, *args, **kwargs):
     return update_wrapper(inner, f)
 
 
-def log_level_option(default=None):
+def log_options(default=None):
     def decorator(f):
-        "Give ability to configure global logging level. Must be used before configuration."
+        """
+        Give ability to configure global logging level/format.
+        Must be used before configuration.
+        """
         import click
         from functools import update_wrapper
+        from sentry.logging import LoggingFormat
+        formats = [LoggingFormat.HUMAN.upper(), LoggingFormat.MACHINE.upper()]
```

Comment:
`CaseInsensitiveChoice` requires all uppercase values for comparison. Anything that you can recommend would probably be better.